web-app/web-frontend/js/chess.js

```diff
@@ -1526,11 +1568,11 @@ var Chess = function (fen) {
             }
             move_string = move_number + '.'
           }
-  
+          
           move_string =
-            move_string +
-            ' ' +
-            move_to_san(move, generate_moves({ legal: true }))
+          move_string +
+          ' ' + 
+          move_to_gan(move, generate_moves({ legal: true }))
```

Comment:
Instead of modifying how the pgn is displayed/saved, can you please display the last move in GAN format in addition to the pgn code that you previously had on the page? i.e. please change this line back to `move_to_san` and add something to script.js and index.html to display the last move in GAN format. Thank you